src/control.py

```diff
@@ -45,61 +106,38 @@ def init_i2c(self):
 
     def init_leak(self):
         self.leak_sensor= AnalogIn(self.ads,ADS.P0)
-        
-    def water_level_init(self):
```

Comment:
The water level sensor does not use the ADC. Please review the recently merged water_level.py on the main branch for code.

Reply:
Sure!